mh_amcl_node:
  ros__parameters:
    use_sim_time: True
    max_particles: 200
    min_particles: 30
    init_pos_x: -2.0
    init_pos_y: -0.3
    init_pos_yaw: 0.0
    init_error_x: 0.1
    init_error_y: 0.1
    init_error_yaw: 0.05
    translation_noise: 0.1
    rotation_noise: 0.05
    distance_perception_error: 0.01
    reseed_percentage_losers: 0.6
    reseed_percentage_winners: 0.05
